oshdb-tool/etl/README.md

```diff
@@ -78,8 +78,8 @@ Apache website.
 
 2. Then start as many ignite servers in a pseudo-distributed system as you
    think your computer can handle using an appropriate ignite-config.xml.
-   Ignite's default configuration file is located at
-   /opt/ignite/config/default-config.xml.
+   Ignites default configuration file is located at
```

Comment:
Is this a false positive when spell checking? :wink:

Reply:
:see_no_evil: :heavy_check_mark: